lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -22,6 +22,8 @@ const Runner = require('../../../lighthouse-core/runner');
 const Config = require('../../../lighthouse-core/config');
 const configJSON = require('../../../lighthouse-core/config/default.json');
 const log = require('../../../lighthouse-core/lib/log');
+const STORAGE_KEY = 'lighthouse_audits';
```

Comment:
Mmh I think a better option to keep all audits in storage with boolean true or false. When we add new keys we can do a simple diff, if key does not exists we put it in as true by default.

So we don't have to care about versioning.